.github/workflows/unit.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.7, 3.8, 3.9]
+        python: [3.7, 3.8, 3.9, '3.10']
```

Comment:
Are the quotes intended?

Reply:
Yes, due to an issue with how the workflows are parsed: https://github.com/actions/setup-python/issues/160